README.md

```diff
@@ -2,4 +2,7 @@
 
 ##SubHeader: My personal tutorial to practice on using Git.
 
-Don't give up and keep practicing.
\ No newline at end of file
+Don't give up and keep practicing.
+
+## Self-Study Concepts
+Languages: Python, Java 8, Golang, C/C++
```

Comment:
This is just a little reminder to take the time to work and practice on these languages. I need to become more proficient on one of these languages.